jekyll/_cci2/storage-lifecycle.adoc

```diff
@@ -0,0 +1,82 @@
+= CircleCI Server AWS S3 Storage Lifecycle
+:page-layout: classic-docs
+:page-liquid:
+:source-highlighter: rouge
+:rouge-linenums-mode: inline
+:icons: font
+:toc: macro
+:toc-title:
+:sectanchors:
+
+
+[.serveronly]_This document is intended for system administrators of self-hosted installations of CircleCI Server._
+
+This guide is intended to help system administrators of CircleCI Server installations on AWS better understand how S3 storage is used. This can help to cut compute resource costs and avoid issues for service users when removing resources.
+
+== Lifecycle Policy Configuration for S3 Buckets
+
+NOTE: Scroll table to the right to view all content.
+
+[.table.table-striped]
+[cols=5*, options="header", stripes=even]
+|===
+| **Prefix**
+| **Tag**
+| **Days**
```

Comment:
Related to the comment above, maybe better to emphasize here that the days are kind-of suggestions, and the numbers are totally up to administrators.